core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/components/ai-image-prompt-form/ai-image-prompt-form.component.ts

```diff
@@ -112,33 +116,38 @@ export class AiImagePromptFormComponent implements OnChanges, OnInit {
 
     private initForm(): void {
         this.form = new FormGroup({
-            text: new FormControl('', Validators.required),
+            text: new FormControl('', [Validators.required, DotValidators.noWhitespace]),
             type: new FormControl(PromptType.INPUT, Validators.required),
             size: new FormControl(DotAIImageOrientation.HORIZONTAL, Validators.required)
         });
 
-        const sizeControl = this.form.get('size');
         const typeControl = this.form.get('type');
 
-        sizeControl.valueChanges.pipe(takeUntilDestroyed(this.destroyRef)).subscribe((size) => {
-            this.orientation.emit(size);
-        });
-
-        typeControl.valueChanges.pipe(takeUntilDestroyed(this.destroyRef)).subscribe((type) => {
-            const promptControl = this.form.get('text');
-            type === PromptType.AUTO
-                ? promptControl.clearValidators()
-                : promptControl.setValidators(Validators.required);
+        this.form.valueChanges
+            .pipe(
+                takeUntilDestroyed(this.destroyRef),
+                filter(() => !this.isUpdatingValidators)
+            )
+            .subscribe((value: AIImagePrompt) => this.valueChange.emit(value));
 
-            promptControl.updateValueAndValidity();
+        typeControl.valueChanges
+            .pipe(takeUntilDestroyed(this.destroyRef))
```

Comment:
`this.destroyRef` is necessary?

Reply:
it is, because is happening in the `ngOnInit`